tinylog-impl/src/test/java/org/tinylog/writers/SyslogWriterTest.java

```diff
@@ -0,0 +1,311 @@
+/*
+ * Copyright 2023 Piotr Karlowicz
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package org.tinylog.writers;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.Test;
+import org.tinylog.Level;
+import org.tinylog.util.LogEntryBuilder;
+import org.tinylog.util.TcpSyslogServer;
+import org.tinylog.util.UdpSyslogServer;
+import org.tinylog.writers.raw.SyslogFacility;
+import org.tinylog.writers.raw.SyslogSeverity;
+
+import static java.util.Collections.emptyMap;
+import static java.util.Collections.singletonMap;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.tinylog.util.Maps.doubletonMap;
+import static org.tinylog.util.Maps.tripletonMap;
+
+/**
+ * Tests for {@link SyslogWriter}.
+ */
+public final class SyslogWriterTest {
```

Comment:
Two general wishes:

1st: If you split lines, please move the dot from the end of the line to the start of the next line.

Example:

```java
assertThatThrownBy(() -> new SyslogWriter(doubletonMap("protocol", "udp", "facility", "invalid"))
		.write(LogEntryBuilder.empty().message(TEST_MESSAGE).create())).hasMessageMatching("(?i).*SyslogFacility.*");
```

2nd: Please add some blank lines in longer test methods to group the logic. This makes it easier to read and unterstand the logic.

Reply:
Updated